README.md

```diff
@@ -48,7 +48,7 @@ TODO
 
 ## Installation 🔧
 
-Alicerce supports iOS 9.0+ and requires Xcode 9.3+.
+Alicerce supports iOS 9.0+ and requires Xcode 10+.
 
```

Comment:
We should probably add a note that for Xcode 9.3 compatibility (Swift 4.0), Alicerce 0.3.0 should be used 💅